react-components/src/architecture/base/concreteCommands/image360Collection/Do360ActionCommand.ts

```diff
@@ -0,0 +1,63 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { Image360Action } from '@cognite/reveal';
+import { RenderTargetCommand } from '../../commands/RenderTargetCommand';
+import { type TranslateKey } from '../../utilities/TranslateKey';
+import { type IconName } from '../../utilities/IconName';
+import { type BaseCommand } from '../../commands/BaseCommand';
+
+export class Do360ActionCommand extends RenderTargetCommand {
```

Comment:
Would prefer have have ore descriptive name, as this command is related to 360 image enter/exit and move forward/backward, we can add rename it as `Image360ActionCommand`